pkg/controllers/resourcechange/resourcechange_controller_test.go

```diff
@@ -49,9 +63,96 @@ func (w *fakeController) Run(_ context.Context, _ int) error {
 }
 
 func (w *fakeController) Enqueue(obj interface{}) {
+	if w.QueueObj == nil {
+		w.QueueObj = []string{}
+	}
 	w.QueueObj = append(w.QueueObj, obj.(string))
 }
 
+// fakeLister is a simple fake lister for testing
```

Comment:
Can we move these to test utils?

Reply:
For now, it's only used by this controller. Will see.